README.md

```diff
@@ -23,8 +23,8 @@ properties are evaluated:
 | [ASReview](#asreview)          | [:link:](https://asreview.nl/)                   | :white_check_mark:[:link:](https://github.com/asreview/)   |:white_check_mark:[:link:](https://doi.org/10.1038/s42256-020-00287-7) |[:link:](https://asreview.readthedocs.io/)                       |   **A**    |:white_check_mark:`+`|
 | [Abstrackr](#abstrackr)        |                                                  | :zap:[:link:](https://github.com/bwallace/abstrackr-web)   |:white_check_mark:[:link:](https://doi.org/10.1145/2110363.2110464)    |[:link:](https://eppi.ioe.ac.uk/)                                |  |  |
 | [Colandr](#colandr)            | [:link:](https://hslib.jabsom.hawaii.edu/colandr)|                                                            |:white_check_mark:[:link:](https://doi.org/10.1111/cobi.13117)         |[:link:](https://hslib.jabsom.hawaii.edu/colandr/getting_started) |  |  |
-| [EPPI-Reviewer](#eppi-reviewer)| [:link:](https://eppi.ioe.ac.uk/cms/)            |                                                            |:white_check_mark:[:link:](https://doi.org/10.1007/s10664-017-9587-0)  |[:link:](https://eppi.ioe.ac.uk/cms/Default.aspx?tabid=3822)     |  |  |
-| [FASTREAD](#fastread)          |                                                  | :white_check_mark:[:link:](https://github.com/fastread/src)|                                                                       |                                                                 |  |  |
+| [EPPI-Reviewer](#eppi-reviewer)| [:link:](https://eppi.ioe.ac.uk/cms/)            | :zap:[:link:](https://doi.org/10.1016/j.eswa.2018.11.021)  |:white_check_mark:[:link:](https://doi.org/10.1007/s10664-017-9587-0)  |[:link:](https://eppi.ioe.ac.uk/cms/Default.aspx?tabid=3822)     |  |  |
```

Comment:
The hyperlink https://doi.org/10.1016/j.eswa.2018.11.021 has been added to the row for EPPI-reviewer. Could you change the hyperlink to a paper introducing EPPI-reviewer? Thanks!

Reply:
I resolved the issue. Added some additional info to Table 1.

Reply:
Many thanks!